src/parser/jobs/rpr/modules/OtherGauges.tsx

```diff
@@ -41,11 +41,18 @@ export class OtherGauges extends CoreGauge {
 
 	// Initialise our gauges - default max is 100 so yolo it is
 	private soulGauge = this.add(new CounterGauge({
-		graph: {label: 'Soul Gauge', color: Color(JOBS.WARRIOR.colour).fade(GAUGE_FADE), collapse: true},
+		graph: {
+			label: <Trans id="rpr.gauge.soulgauge.label">Soul Gauge</Trans>,
+			color: Color(JOBS.WARRIOR.colour).fade(GAUGE_FADE),
+			collapse: false,
```

Comment:
Is `collapse: false` here intentional? I don't have a preference but currently the two MCH gauges are collapsed by default, and if we're changing it here then I'd prefer for it to be changed for MCH as well for consistency's sake.

Reply:
I don't remember, but I like the way it looks better, so sure, we can do it for MCH (and DNC actually) too. There's a few other jobs that are still defaulting collapsed, but I'd want their maintainers to make the decision before mucking with them so I'll leave them be for now.